_posts/2019-06-27-list-strategy.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: posts
+title: "List based execution strategy"
+authors:
+  - ishikhman
+---
+**tl;dr**: changes are coming with bazel 0.27 release: bazel supports list based execution strategy; build will fail if action cannot be executed with provided list of strategies; bazel will *auto-detect* a suitable strategy if none provided.
+
+## How it was before bazel 0.27 release
```

Comment:
I think this paragraph might be more effective to discuss what problem this solves:
1. Bazel's defaults did not account for remote execution at all and so we had to make sure to set the execution strategy for every mnemonic manually.
2. Starlark rules providing a persistent worker had to ship a `.bazelrc` file that sets `--strategy=Mnemonic=worker` (i.e. look at rules_scala).

Both of the above are now unnecessary with this change.